### PR TITLE
Middle Section of Portfolio project

### PR DESCRIPTION
**Pull request by AFakhriddinov**
**In this milestone,**
Continued to build existing portfolio project
Created a new branch called "middle_section"
Used grid layout and flexbox layout
Made pages responsive with grid and flexbox styling
Added "Works" content
Added "About me" content
Tested for linter errors
Pushed to GitHub branch called "middle_section"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,5 +81,212 @@ <h1>
...
+            <li class="platform">
+              <a href="#html">html</a>
+            </li>
+            <li class="platform">
+              <a href="#CSS">css</a>
+            </li>
+            <li class="platform">
+              <a href="#JS">javascript</a>
+            </li>
+          </ul>
```
Comment (joel-kalema) [inline index.html]: Good job so far 😃, As I see you made great work, but  you need to fix some parts 👍 

- [x] Kindly make sure you fixed the displaying of the languages list to stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |cart01| |cart| |

File: index.html [hunk 2]
```diff
@@ -81,5 +81,212 @@ <h1>
...
+      </div>
+    </section>
+    <!-- About me section -->
+    <section class="about-me">
+      <div class="page">
+        <h2 class="head">
+          About
+          <br />
+          Myself
+        </h2>
```
Comment (joel-kalema) [inline index.html:216]: - [x] Also, please fix the `margin`, the `color` and the `font size`  to stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍
> The h2 font-size: 40px;
> The h2 color: #172B4D;

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |mob01| |mob| |

File: index.html [hunk 3]
```diff
@@ -81,5 +81,212 @@ <h1>
...
+          <button type="button" class="content-view-button">See Project</button>
+        </div>
+      </div>
+      <div class="content">
+        <img src="imgages/project2.svg" alt="Work 2" class="content-picture" />
+        <div class="content-body">
+          <h2 class="content-title">
+            Multi-Post <br />
+            Stories
+          </h2>
```
Comment (joel-kalema) [inline index.html]: - [x] Also, there is a broken link image, Kindly fix this part 👍 

cart02

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @AFakhriddinov 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 

- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -81,5 +81,212 @@ <h1>
...
+      </div>
+    </section>
+    <!-- About me section -->
+    <section class="about-me">
+      <div class="page">
+        <h2 class="head">
+          About
+          <br />
+          Myself
+        </h2>
```
Comment (DuaneDave) [inline index.html:216]: As previously [requested](https://github.com/AFakhriddinov/portfolio-mobile-first/pull/3/files#r1115514897), kindly add some margin to the 'About Myself' text to push it away from the top of the parent container in order to make it match the Figma template.

After testing, I found that a margin-top of 4rem would fix the issue

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @AFakhriddinov 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ The UI looks awesome
✔️ Great PR summary
✔️ Correct usage of GitHub flow


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @AFakhriddinov , 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Good design ✅
- Good use of flexbox and grid ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**